R/ForecastRegrTask.R

```diff
@@ -0,0 +1,58 @@
+#' @title Create a task for univariate forecasting
+#'
+#' @rdname Task
+#'
+#' @description Creates a task for univariate forecasting learners
+#'
+#' @export
+makeForecastRegrTask = function(id = deparse(substitute(data)), data, target,
+  weights = NULL, blocking = NULL, frequency = 1L, fixup.data = "warn",
+  check.data = TRUE) {
+  assertString(id)
+  assertClass(data, "xts")
```

Comment:
The more I think about this my only arguments against this now are that I'm lazy and I have a suspicion in the back of my mind this would be more involved than I am seeing right now. Let me mess with this and see if I can find something satisfying